src/lib/color.ts

```diff
@@ -0,0 +1,10 @@
+import { stdout } from "supports-color";
+
+// The value for the `color` parameter to the `/schema/1` endpoints.
+export const colorParam = (): string => {
+  return hasColor() ? "ansi" : "";
+};
+
+export const hasColor = (): boolean => {
+  return stdout !== false && stdout.hasBasic;
```

Comment:
`supports-color` defines 4 levels of color:
- no support
- basic support (0-16 basic ansi colors)
- 256 color support
- full color support

Core only emits the basic 0-16 ansi colors, so we just check for that.